src/backend/server.ts

```diff
@@ -83,6 +85,8 @@ app.use(
     doProxy(ApplicationName.klage)
 );
 
+configureToggleEndpoint(app);
```

Comment:
Foreslår at vi bruker samme struktur her som i linjene over. Altså,

```suggestion
app.use(
    '/api/toggle',
    addRequestInfo(),
    validateToken(),
    attachUnleashAuthToken(),
    doProxy(ApplicationName.unleash)
);
```